backend/campaigns/models.py

```
@@ -0,0 +1,288 @@
+    DEVICE_DESKTOP = 'desktop'
+    DEVICE_MOBILE = 'mobile'
+    DEVICE_TABLET = 'tablet'
+    DEVICE_BOT = 'bot'
+    DEVICE_UNKNOWN = 'unknown'
+    DEVICE_CHOICES = [
+        (DEVICE_DESKTOP, 'Desktop'),
+        (DEVICE_MOBILE, 'Mobile'),
+        (DEVICE_TABLET, 'Tablet'),
+        (DEVICE_BOT, 'Bot'),
+        (DEVICE_UNKNOWN, 'Unknown'),
+    ]
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)